packages/hardhat/contracts/Ethernauts.sol

```diff
@@ -27,7 +27,6 @@ contract Ethernauts is ERC721Enumerable, Ownable, ReentrancyGuard {
     error SaleStateError(SaleState newState, SaleState current);
     error RecoverTokenError(address tokenAddress, address toAddress);
     error TokenBalanceError(uint256 tokenBalance, uint256 amount);
-    error TotalSupplyError(uint256 total, uint256 max);
```

Comment:
Removing this error, not because I don't wanna test it =P but because the error is actually unreachable (other errors will be thrown before this one).